packages/plugins/typescript/typescript/src/config.ts

```diff
@@ -84,7 +84,7 @@ export interface TypeScriptPluginConfig extends RawTypesConfig {
    */
   numericEnums?: boolean;
   /**
-   * @description This option controls whether or not a catch-all entry is added to enum type definitions for values that may be added in the future. You also have to set `enumsAsTypes` to true if you wish to use this option.
```

Comment:
The removed sentence is no longer true. Should we go into details how this behavior is different for `enumsAsTypes` though?